### PR TITLE
feat(bindings): expose getting member by id

### PR DESCRIPTION
**Pull request by csmith**
Allow for retrieving a single room member by their ID.

### CODE REVIEW

File: bindings/matrix-sdk-ffi/src/room.rs [hunk 1]
```diff
@@ -187,6 +187,16 @@ impl Room {
         })
     }
 
+    pub fn member(&self, user_id: String) -> Result<Arc<RoomMember>, ClientError> {
+        let room = self.room.clone();
+        let user_id = user_id;
+        RUNTIME.block_on(async move {
+            let user_id = <&UserId>::try_from(&*user_id).context("Invalid user id.")?;
```
Comment (poljar) [inline bindings/matrix-sdk-ffi/src/room.rs:194]: [`UserId::parse()`](https://docs.rs/ruma-identifiers/latest/ruma_identifiers/struct.UserId.html#method.parse) is a bit more pleasant to look at. Let's use it.

Comment (jplatte) [reply bindings/matrix-sdk-ffi/src/room.rs:194]: Well, that one allocates which `<&UserId>::try_from` doesn't ^^
But maybe it's not all that relevant.

**Comment by csmith** [comment]
@poljar friendly ping :)

**Comment by poljar** [comment]
> @poljar friendly ping :)

Gah, sorry.

**Review by poljar — APPROVED** [review]

**Pull request merged**